content/docs/04-clusters/02-data-center/02-openstack.md

```diff
@@ -361,16 +362,239 @@ The following prerequisites must be met before deploying a Kubernetes clusters i
+
+<br />
+
+
+## Validate
+
+Once installed, the PCG registers itself with Palette. To verify the PCG is registered, use the following steps.
+
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+
+2. Navigate to the left **Main Menu** and select **Tenant Settings**
+
+
+3. From the **Tenant Settings Menu** click on **Private Cloud Gateways**. Verify your PCG cluster is available from the list of PCG clusters displayed.
+
+
+4. When you install the PCG, a cloud account is auto-created. To verify the cloud account is created, go to **Tenant Settings > Cloud Accounts** and locate **OpenStack** in the table. Verify your OpenStack account is listed.
+
+
+
+## Edit and Redeploy PCG
+
+To change the PCG install values, restart the installation process using the `palette pcg install` command.  Use the following steps to redeploy the PCG or restart the install process. 
+
+<br />
+
+1. Make the necessary changes to the PCG configuration file the CLI created during the installation, if needed. Use a text editor, such as vi or nano to update the PCG install configuration file.
+
+  <br />
+
+  ```shell hideClipboard
+  ==== Create PCG reference config ====
+  ==== PCG config saved ====
+  Location: /Users/demo/.palette/pcg/pcg-20230717114807/pcg.yaml
+  ```
+
+  ```bash hideClipboard
+  vi /home/demo/.palette/pcg/pcg-20230706150945/pcg.yaml
+  ```
+
+
+
+2. To redeploy the PCG, use the `install` command with the flags `--silent` and `--config-file`. Provide the file path to the generated PCG config file that was generated and displayed in the output. 
+
+  <br />
+
+  ```bash hideClipboard
+  palette pcg install --silent --config-file /home/demo/.palette/pcg/pcg-20230706150945/pcg.yaml
+  ```
+
+</Tabs.TabPane>
+
+
+
+<Tabs.TabPane tab="PCG Installer Image" key="pcg-installer-image">
+
+
  `video: title: "openstack-pcg-creation": /pcg-creation-video/openstack.mp4`
 
-The following system requirements should be met in order to install a private cloud gateway for OpenStack:
 
-* Private cloud gateway IP requirements:
-    * 1 IP for a 1 node PCG or 3 IPs for a 3 node PCG
-    * 1 IP for Kubernetes control-plane
 
-Palette provides an installer in the form of a docker container. This installer can be run on any system that has docker daemon installed and has connectivity to the Palette Management console as well as OpenStack controller.
+## Prerequisites
+
+The following system requirements are required to deploy a PCG cluster.
 
+- Palette version 3.4.X or older. 
+
+- A Linux environment with a Docker daemon installed and a connection to Palette and the OpenStack environment. The installer must be invoked on an up-to-date Linux system with an x86-64 architecture. ARM architecture is currently not supported.
+
+
+- Private cloud gateway IP requirements:
```

Comment:
```suggestion
- Private Cloud Gateway IP requirements:
```

Reply:
I can't tell if PCG has previously been spelled out in this doc. If not, it should at least be initial capped. Since PCG appears in line 588, maybe this should also be PCG.